src/core/packages/chrome/browser-internal/src/ui/header/__snapshots__/collapsible_nav.test.tsx.snap

```diff
@@ -863,29 +863,105 @@ exports[`CollapsibleNav renders the default nav 1`] = `
   <EuiCollapsibleNav
     aria-label="Primary"
     button={<button />}
-    css={
-      Object {
-        "map": undefined,
-        "name": "1pvcuvk",
-        "next": undefined,
-        "styles": "@media (max-height: 240px){overflow-y:auto;}",
-        "toString": [Function],
-      }
-    }
+    css="unknown styles"
     data-test-subj="collapsibleNav"
     id="collapsibe-nav"
     isOpen={false}
     onClose={[Function]}
     ownFocus={false}
     size={248}
   >
-    <button
-      aria-controls="collapsibe-nav"
-      aria-expanded={false}
-      aria-pressed={false}
-      onMouseUpCapture={[Function]}
-      onTouchEnd={[Function]}
+    <Insertion
```

Comment:
These enzyme snapshot changes are super ugly. I tried to use https://emotion.sh/docs/@emotion/jest @emotion/jest/enzyme-serializer but it didn't work (runtime errors). I assume those are no longer supported